AirCasting/CreateSessionViews/SelectDeviceView.swift

```diff
@@ -16,12 +16,13 @@ struct SelectDeviceView: View {
     @State private var isTurnOnBluetoothLinkActive: Bool = false
     @State private var isPowerABLinkActive: Bool = false
     @State private var isMicLinkActive: Bool = false
+    @State private var isBTButtonAvailable: Bool = false
```

Comment:
small digression: "is" prefix is not needed for this kind of properties, see: `btButtonAvailable: bool` says it all already. Ofc don't change it